spec/config/ap.yml

```diff
@@ -16,13 +16,10 @@
 #  either give absolute path or path relative to where the command will be executed
 #whitelist: config/ap_whitelist.txt
 
-dor_fetcher:
-  # the baseurl of the DOR Fetcher service from which we get the item druids (per whitelist above)
-  #  do not include 'collections' at end.
-  service_url: http://127.0.0.1:3000
-  # if skip_heartbeat set to true, this will skip a check that the dorfetcher service is alive before making API calls
-  #   (useful for testing)
-  skip_heartbeat: true
+dor_services:
+  # the baseurl of the dor-services-app from which we get the item druids (per whitelist above)
+  url: https://dor-services-stage.stanford.edu
+  token: eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIn0.dozjgNryP4J3jVmNHl0w5N_XgL0n3I9PlFUP0THsR8U
```

Comment:
@jcoyne Should we care about having a stage JWT checked in?

Reply:
that isn't signed by our stage server.